api/graphql/tests/test_reservation_units.py

```diff
@@ -1113,6 +969,349 @@ def test_getting_terms(self):
         self.assertMatchSnapshot(content)
 
 
+class ReservationUnitsFilterTextSearchTestCase(ReservationUnitQueryTestCaseBase):
+    def test_filtering_by_type_fi(self):
+        response = self.query(
+            """
+            query {
+                reservationUnits(textSearch:"Test type fi"){
+                edges {
+                    node {
+                        nameFi
+                        reservationUnitType {
+                            nameFi
+                        }
+                    }
+                }
```

Comment:
Maybe we could indent these lines 2 spaces more to make the indentation consistent. (Also in the other queries below.

Reply:
Yes, would be more readable :+1: